installer/lib/mix/tasks/phx.new.ex

```diff
@@ -49,6 +49,9 @@ defmodule Mix.Tasks.Phx.New do
     * `--binary-id` - use `binary_id` as primary key type
       in Ecto schemas
 
+    * `--no-install` - do not fetch and install dependencies
+      while generating this project
```

Comment:
Given the default is `--install`, we only document the non-default behaviour. :)

Reply:
Oh, i see though. You are making it so it doesn't even ask. Ignore me, i will add it back.